src/testkit/new/Component.driver.ts

```diff
@@ -1,8 +1,8 @@
 import {ReactTestInstance} from 'react-test-renderer';
-import {RenderResult} from '@testing-library/react-native';
+import {within} from '@testing-library/react-native';
 
 export interface ComponentProps {
-  renderTree: RenderResult;
+  renderTree: ReturnType<typeof within>;
```

Comment:
Since it looks strange, I would add a `// Notes:` that explain in short why it was necessary

Reply:
Added.